### PR TITLE
New package: JetBrains DataSpell 2022.3.2

### PR DESCRIPTION
**Pull request by 0xr0bert**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98136)

### CODE REVIEW

**Comment by 0xr0bert** [comment]
@microsoft-github-policy-service agree

File: manifests/j/JetBrains/DataSpell/2022.3.2/JetBrains.DataSpell.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: JetBrains.DataSpell
+PackageVersion: 2022.3.2
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
```
Comment (mdanish-kh) [inline manifests/j/JetBrains/DataSpell/2022.3.2/JetBrains.DataSpell.installer.yaml:9]:
```suggestion
- silent
AppsAndFeaturesEntries:
- DisplayVersion: 223.8617.58
  ProductCode: DataSpell 2022.3.2
```

To match registry entries

File: manifests/j/JetBrains/DataSpell/2022.3.2/JetBrains.DataSpell.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: JetBrains.DataSpell
+PackageVersion: 2022.3.2
+PackageLocale: en-US
+Publisher: JetBrains s.r.o.
+PublisherUrl: https://www.jetbrains.com
+PublisherSupportUrl: https://www.jetbrains.com/data-tools
+PrivacyUrl: https://www.jetbrains.com/privacy-security
+PackageName: JetBrains DataSpell
```
Comment (mdanish-kh) [inline manifests/j/JetBrains/DataSpell/2022.3.2/JetBrains.DataSpell.locale.en-US.yaml:11]:
```suggestion
PackageName: DataSpell
```
Must match Control Panel / Apps&Features

File: manifests/j/JetBrains/DataSpell/2022.3.2/JetBrains.DataSpell.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+PackageUrl: https://www.jetbrains.com/dataspell
+License: Proprietary
+Copyright: Copyright (c) JetBrains s.r.o.
+CopyrightUrl: https://www.jetbrains.com/legal/#licensing
+ShortDescription: A new IDE by JetBrains for data science.
+Moniker: dataspell-preview
```
Comment (mdanish-kh) [inline manifests/j/JetBrains/DataSpell/2022.3.2/JetBrains.DataSpell.locale.en-US.yaml:17]: Is the package still in Preview? If so, then consider adding it under [JetBrains.DataSpell.EarlyPreview](https://github.com/microsoft/winget-pkgs/tree/master/manifests/j/JetBrains/DataSpell/EarlyPreview) where other versions already exist.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/j/JetBrains/DataSpell/2022.3.2/JetBrains.DataSpell.installer.yaml (hunk 1 above)
Comment (0xr0bert) [reply manifests/j/JetBrains/DataSpell/2022.3.2/JetBrains.DataSpell.installer.yaml:9]: Thanks

File: manifests/j/JetBrains/DataSpell/2022.3.2/JetBrains.DataSpell.locale.en-US.yaml (hunk 3 above)
Comment (0xr0bert) [reply manifests/j/JetBrains/DataSpell/2022.3.2/JetBrains.DataSpell.locale.en-US.yaml]: No, sorry, I just copied the manifest for the preview to add the non-preview. Have fixed

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**